all_test.go

```diff
@@ -372,8 +386,13 @@ func TestOptions_CopyRateLimit(t *testing.T) {
 	start := time.Now()
 	err = Copy("test/data/case16", "test/data.copy/case16", opt)
 	elapsed := time.Since(start)
-	Expect(t, err).ToBe(nil)
-	Expect(t, elapsed > 5*time.Second).ToBe(true)
+	if supportsWrapReaderOption {
```

Comment:
I totally disagree with this `if`.
If I were a user reading this test code,

1. first I would think "hmm, what's this `supportsWrapReaderOption`?"
2. second I would search `supportsWrapReaderOption` and find `setupFileCopyMethod()` and would think "now, what's `TEST_FILECOPYMETHOD`?"
3. third, I would find `${{ matrix.os.copymethod }}` then I would know it's deeply coupled with filesystem

It is a long journey. We need to find more straightforward way so that users can easily understand.

Reply:
I could move `supportsWrapReadOption` into the `FileCopyMethod` struct? That might be better.